terraform/config.tf

```diff
@@ -17,6 +17,7 @@ provider "aws" {
 
 
 provider "fastly" {
+  version = "~> 0.3.0"
```

Comment:
pinned until we sort out how we'll handle snippets now that fastly provider can do so